pennylane/devices/default_qubit.py

```diff
@@ -26,6 +26,7 @@
 from scipy.sparse import coo_matrix
 
 import pennylane as qml
+import jax
```

Comment:
Best not to import soft dependencies like JAX here.

Because it is likely that some users will not have JAX installed locally, this will cause `import pennylane as qml` to fail with an import error!